api/src/main/resources/public/api/v1/queue.raml

```diff
@@ -0,0 +1,10 @@
+get:
+  description:
+    Get the list of all jobs in the launch queue
+  is: [ secured ]
+  responses:
+    200:
+      description: "The list of jobs in the launch queue."
```

Comment:
I think we can lose the " here

Reply:
we have quotes in other another raml as well... I will remove it in all ramls for consistency